mink/lie/base.py

```diff
@@ -103,6 +103,21 @@ def normalize(self) -> Self:
         """Normalize/projects values and returns."""
         raise NotImplementedError
 
+    def interpolate(self, other: Self, alpha: float = 0.5) -> Self:
+        """Interpolate between two matrix Lie groups.
+
+        Args:
+            other: The other Lie group, which serves as the end point of interpolation.
+            alpha: The fraction of interpolation between [self, other]. This must
+                be within [0.0, 1.0]. 0.0 = self, 1.0 = other.
+
+        Returns:
+            The interpolated matrix Lie group.
+        """
+        if alpha < 0.0 or alpha > 1.0:
+            raise ValueError(f"Expected alpha within [0.0, 1.0] but received {alpha}")
+        return self @ self.exp(alpha * (self.inverse() @ other).log())
+
     # Plus and minus operators.
 
     # Eqn. 25.
```

Comment:
I've seen this throughout the codebase, is this an internal equation tracker or is it referring to an external resource? Do I need to add something similar for the `interpolate` method?

Reply:
Nah that’s just referencing the equations in the [microlie](https://arxiv.org/abs/1812.01537) paper. Was useful as I was re-implementing the methods.